Python/sort/Parallel_MergeSort.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+""" Solution: Sort an array of random integers with merge sort using parallel approach."""

```

Comment:
@PinkyWalia update the description of the program since both parallel and sequential merge sort is used.